.circleci/config.yml

```diff
@@ -379,8 +385,12 @@ workflows:
       - cpp-gcc:
           matrix:
             parameters:
-              gcc-version: ["11", "12", "13", "14"]
-      - cpp-macOS
+              gcc-version: ["11", "12", "13", "14", "15"]
+              cpp-standard: ["default", "c++23"]
```

Comment:
I understand you're trying to test that the default works, but IMO we should just explicitly sweep compiler versions. It's simpler and I think this will rarely change.
```
             cpp-standard: ["c++20", "c++23"]
```